org-roam.el

```diff
@@ -1227,7 +1236,9 @@ If item at point is not Org-roam specific, default to Org behaviour."
          (cons '(file . org-roam--find-file) org-link-frame-setup))
         (let ((inhibit-read-only t))
           (erase-buffer)
-          (unless (eq major-mode 'org-roam-backlinks-mode)
+          (unless (eq major-mode 'org-mode)
+            (org-mode))
+          (unless org-roam-backlinks-mode
```

Comment:
This gated activation of the major and minor mode has the effect that the hooks are not run each time, which I think is intentional? In any case, I tried to keep the current behavior there.